helm/values/dev-values.yaml

```diff
@@ -22,3 +22,10 @@ accruals:
     name: callisto-accruals-restapi
     port: 8080
   host: accruals.dev.callisto-notprod.homeoffice.gov.uk
+
+person:
+  name: callisto-person-restapi
+  service:
+    name: callisto-person-restapi
+    port: 8080
```

Comment:
Is the port set to 9090 in person service?

Reply:
Correct. Amended